CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp-main.tex

```diff
@@ -416,7 +417,8 @@ \section{Software and Computing
 provide practical solutions to ours. The efficient processing of massive
 data volumes is also a challenge faced by industry, in particular the
 internet economy, which developed novel and major new technologies under
-the banner of \emph{Big Data} that may be applicable to our use cases.
+the banner of \emph{Big Data} that may be applicable to our use cases
+\cite{47224,Shanahan:2015:LSD:2783258.2789993,Armbrust:2015:SSR:2723372.2742797}.
```

Comment:
Seems the last reference "Armbrust:2015:SSR:2723372.2742797" has not been added to the bib file.

Reply:
It was already there for a long time:

```
0e2ac222 CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib (Oliver Gutsche    2018-02-01 07:37:42 -0600 2226) @inproceedings{Armbrust:2015:SSR:2723372.2742797,
```

Reply:
Oh, I managed to miss it. Sorted, then!